src/Symfony/Bundle/FrameworkBundle/Client.php

```diff
@@ -104,6 +106,34 @@ public function enableReboot()
         $this->reboot = true;
     }
 
+    /**
+     * Logins a user against a specific firewall.
+     *
+     * @param string   $username
+     * @param string   $password
+     * @param string   $firewall
+     * @param string[] $roles
+     */
+    public function login($username, $password, $firewall = 'main', $roles = array('ROLE_ADMIN'))
```

Comment:
We can add PHP 7 type-hints to all these arguments.

Reply:
Fixed